### PR TITLE
feat: add lottie alpha property & export isPlaying & play continue & stop

### PR DESCRIPTION
**Pull request by YunYouJun**
- add global alpha (support playing updateElement)
- optimize example (add play/alpha options)
- export `isPlaying` state (it's useful)
- feat: add play continue & stop

### CODE REVIEW

**Comment by YunYouJun** [comment]
Kapture 2023-07-03 at 12 11 18

File: src/LottieAnimation.ts [hunk 1]
```diff
@@ -77,6 +81,29 @@ export class LottieAnimation extends Script {
...
+  set alpha(value: number) {
+    // update in updateElement
+    if (this._alpha !== value) {
+      this._alpha = value;
+
+      const isPlaying = this.isPlaying;
+      if (!isPlaying) {
+        this.play();
+      }
+      this.onUpdate(0);
```
Comment (singlecoder) [inline src/LottieAnimation.ts:96]: 不要手动去掉用 onUpdate

Comment (YunYouJun) [reply src/LottieAnimation.ts:96]: 调用 onUpdate 更新子元素 alpha，这里参照的 set resource 逻辑

File: src/LottieAnimation.ts [hunk 2]
```diff
@@ -77,6 +81,29 @@ export class LottieAnimation extends Script {
...
+  /**
+   * global alpha
+   */
+  set alpha(value: number) {
+    // update in updateElement
+    if (this._alpha !== value) {
+      this._alpha = value;
+
+      const isPlaying = this.isPlaying;
+      if (!isPlaying) {
```
[the remark this answers is not in the document]
Comment (YunYouJun) [reply src/LottieAnimation.ts:93]: 如果动画未播放，修改透明度没有即时显示效果

**Review by eyworldwide — APPROVED** [review]

**Comment by YunYouJun** [comment]
> @YunYouJun Thanks for your PR. Please update the version of package for NPM publishing.

Done.

**Pull request merged**